docs/config.rst

```diff
@@ -447,25 +495,6 @@ Features
 
    .. versionadded:: 2.2
 
-.. attribute:: LP_ENABLE_ENV_VARS
-   :type: bool
-   :value: 1
-
-   Display a user-defined set of environment variables.
-   May show if the variables are unset, set, or their actual content.
-
-   Watched variables should be added to the :attr:`LP_ENV_VARS` array.
-
-   The resulting prompt section is configured by:
-
-   - :attr:`LP_MARK_ENV_VARS_OPEN`
-   - :attr:`LP_MARK_ENV_VARS_SEP`
-   - :attr:`LP_MARK_ENV_VARS_CLOSE`
-   - :attr:`LP_COLOR_ENV_VARS_SET`
-   - :attr:`LP_COLOR_ENV_VARS_UNSET`
-
-   .. versionadded:: 2.2
-
```

Comment:
I think your rebase messed something up. Large chunks of this file have been removed.

Reply:
What a mess. I have no idea what happened.

Reply:
I did a revert commit, so as to avoid a rebase during the review.